src/trace/log.zig

```diff
@@ -253,114 +240,228 @@ pub const StandardErrLogger = struct {
 /// where some log messages never get logged because the logger is deinitialized before the
 /// logging thread picks up the log message.
 pub const TestLogger = struct {
```

Comment:
Indeed. `DirectPrintLogger`, and `ChannelPrintLogger` are more descriptive. And after this is merged, I plan to make the `writer` configurable, so it would be possible to log to a file or to standard err etc, and for that the `StandardErrLogger` will no longer apply.

Reply:
Done in [dc73e6d](https://github.com/Syndica/sig/pull/277/commits/dc73e6d7227917c27f43ff7eb5269489d7c89cc1)